src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -369,14 +382,22 @@ private void setupHosting(Channel channel, User sender, String hostingId, String
             String defaultAssignee = reporter != null ? reporter.getName() : "";
 
             String forkFrom = JiraHelper.getFieldValueOrDefault(issue, FORK_FROM_JIRA_FIELD, "");
-            String userList = JiraHelper.getFieldValueOrDefault(issue, USER_LIST_JIRA_FIELD, "");
-            for (String u : userList.split("\\n")) {
+            String githubUserList = JiraHelper.getFieldValueOrDefault(issue, GITHUB_USER_LIST_JIRA_FIELD, "");
+            for (String u : githubUserList.split("\\n")) {
                 if(StringUtils.isNotBlank(u))
-                    users.add(u.trim());
+                    githubUsers.add(u.trim());
             }
             String forkTo = JiraHelper.getFieldValueOrDefault(issue, FORK_TO_JIRA_FIELD, defaultForkTo);
 
-            if(StringUtils.isBlank(forkFrom) || StringUtils.isBlank(forkTo) || users.isEmpty()) {
+            String releaseUserList = JiraHelper.getFieldValueOrDefault(issue, RELEASE_USER_LIST_JIRA_FIELD, "");
+            for (String u : releaseUserList.split("\\n")) {
```

Comment:
While this does not exist, a sane default is the user name who filed the HOSTING issue.

Reply:
I totally missed your feedback. I need to get back to this.